{"name": "test_login_noExistUser_wrongPasswd[data1]", "status": "passed", "attachments": [{"name": "log", "source": "ca265e5b-8aad-48b4-b270-370136f3e9fd-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'用户名': 'dsadsadsa', '密码': 'pytho', '预期': '请输入至少6位数的密码'}"}], "start": 1587024219344, "stop": 1587024220219, "uuid": "5b5d1f88-4061-4eda-b38e-2dac70ced8b0", "historyId": "634fa2f876868d6c8f81ec03e35c966f", "testCaseId": "d35b8259a99be3473adfcaff1df8682d", "fullName": "TestCases.test_00_login.TestLogin#test_login_noExistUser_wrongPasswd", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_00_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-O2A05I4"}, {"name": "thread", "value": "15404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_00_login"}]}